cmd/podman/main_local.go

```diff
@@ -243,6 +243,27 @@ func setupRootless(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func useUserNS(cmd *cobra.Command) bool {
+	if os.Geteuid() == 0 {
+		return false
+	}
+	switch cmd {
+	case _migrateCommand,
+		_migrateCommand,
+		_mountCommand,
+		_renumberCommand,
+		_infoCommand,
+		_searchCommand,
+		_systemResetCommand,
+		_versionCommand:
+		return false
+	}
+	if strings.HasPrefix(cmd.Use, "help") {
+		return false
+	}
+	return true
```

Comment:
```suggestion
	return !strings.HasPrefix(cmd.Use, "help")
```

Reply:
Can you add a comment to this check? As far as I understand it, we are checking if the "Use" string of the command starts with "help" but there are no such occurrences in the code:
```bash
$ grep "Use:" -r cmd/ | grep help | wc -l
0
```

Reply:
I think this was added for commands like

podman run --help

Reply:
Done.

Reply:
`Cmd.Use` is the wrong field for that:
```golang
// Command is just that, a command for your application.              
// E.g.  'go run ...' - 'run' is the command. Cobra requires          
// you to define the usage and description as part of your command    
// definition to ensure usability.                                    
type Command struct {                                                 
        // Use is the one-line usage message.                         
        Use string                                                    
```

Reply:
I printf-debuged this code path. If we run `--help` we don't execute the code path. I guess we can just `return true`.

Reply:
Nice, so I guess this code is not needed, removed.